src/services/event-producer/sqs/sqs-event-producer-service.ts

```diff
@@ -0,0 +1,34 @@
+import 'reflect-metadata'
+import { Config } from 'node-config-ts'
+import { inject, singleton } from 'tsyringe'
+import { SQSClient, SendMessageCommand } from '@aws-sdk/client-sqs'
+import { EventProducerService } from '../event-producer-service.js'
+import { v4 as uuidv4 } from 'uuid'
+
+@singleton()
+export class SQSEventProducerService implements EventProducerService {
+  private readonly sqsClient: SQSClient
+  constructor(@inject('config') private config?: Config) {
+    this.sqsClient = new SQSClient({})
+  }
+
+  /**
+   * Emits an anchor event by sending a message to the configured SQS
+   */
+  public async emitAnchorEvent(): Promise<void> {
+    await this.sqsClient.send(
```

Comment:
Might be good to add a to-do here for later to switch to localstack for unit tests. Even though the current approach was my recommendation, I don't quite like the idea of a unit test environment that's not fully self-contained, and has to reach out to AWS for something. That just adds a whole host of dependencies for anyone trying to run the unit tests, like credentials, etc.